go.mod

```diff
@@ -16,6 +17,7 @@ require (
 	github.com/operator-framework/operator-sdk v0.19.4
 	github.com/sergi/go-diff v1.1.0 // indirect
 	github.com/sirupsen/logrus v1.6.0 // indirect
+	github.com/tidwall/gjson v1.6.3
```

Comment:
i'm still unclear why we're importing this when we have `encoding/json` built in

Reply:
Peri has given explanation https://github.com/openshift/elasticsearch-operator/pull/583#issuecomment-742361069

Reply:
ack, i've seen his response... i guess my question is more, do we really get much from there vs what we could get just from `encoding/json`? To me it seems like an unnecessary library to vendor in

Reply:
Ok I think may be type safety is the advantage mentioned by peri. Also I find code is more readable. If we use encoding then we would again need to use walkInterfaceMap for parsing and other libraries to convert values in boolean etc.  tidewall/json provides parsing and type safety.  Reverting back will require testing again :)

Reply:
part of the reason we are moving to using the elasticsearch api is so that we can use their objects to unmarshall into. We should not need to worry about type safety or walking the interface map in that case.

If they do not have api objects for us the unmarshall into, I feel that this PR doesn't bring us much value over what we currently have...

Reply:
Ok in that case I will look at encoding/json API. I think I will require changes at multiple places so will take some time. Also it will be helpful if  In the meanwhile you can test in your local cluster to validate everything else is working fine.